account_brand/models/account_move.py

```diff
@@ -20,7 +20,7 @@ class AccountMove(models.Model):
 
     def _is_brand_required(self):
         self.ensure_one()
-        if self.type in ("in_invoice", "in_refund"):
+        if self.type in ("in_invoice", "in_refund", "entry"):
             return False
         return super()._is_brand_required()
```

Comment:
Let be exclusive to prevent the same issue on other types
```suggestion
        if self.type in ("out_invoice", "out_refund"):
            return super()._is_brand_required()
        return False
```

now, we should add the move type in unit tests: https://github.com/OCA/brand/blob/13.0/account_brand/tests/test_brand_mixin.py#L23